addon/templates/components/polaris-popover.hbs

```diff
@@ -1,6 +1,7 @@
 {{#basic-dropdown
   verticalPosition=verticalPosition
   onOpen=(action "onOpen")
+  onClose=onClose
```

Comment:
Wrap this in an `action` helper as well to ensure context is always correctly maintained (it's what @vladucu would want 😉)

Reply:
yes, please...it's also much easier to read 😄